### DISCUSSION TITLE
DBeaver to Redshift connection error iam

### DISCUSSION THREAD

**Discussion by faige78** [dbeaver/dbeaver#22891] [Q&A] [answered]
Hi Gurus,

Was trying to setup redshift SSO using DBeaver. And i am encountering this error
com.amazonaws.AmazonClientException

It was fine before where i was logging in using DB user and password

**Comment by LonwoLonwo** [comment]
Hello @faige78 

What DBeaver version do you use?

How exactly are you using SSO authentication in DBeaver?

**Reply by faige78** [reply, marked as answer]
Am using 22.2.2xxxx
I seem to have solve my own problem. For some reason, jdbc driver was not in the original folder. I simply repoint to the correct location

thanks
